spark/src/main/scala/org/apache/comet/serde/QueryPlanSerde.scala

```diff
@@ -1396,6 +1397,16 @@ object QueryPlanSerde extends Logging with ShimQueryPlanSerde {
           val optExpr = scalarExprToProto("atan2", leftExpr, rightExpr)
           optExprWithInfo(optExpr, expr, left, right)
 
+        case e: Unhex if !isSpark32 =>
```

Comment:
Unrelated, but potentially we can vote in the community when to deprecate 3.2 support...